src/installer/corehost/cli/hostpolicy/hostpolicy.cpp

```diff
@@ -744,7 +745,7 @@ SHARED_API int HOSTPOLICY_CALLTYPE corehost_initialize(const corehost_initialize
             rc = StatusCode::Success_DifferentRuntimeProperties;
     }
 
-    auto version_lo = context_contract->version;
+    auto version_lo = version_set ? context_contract->version : 7 * sizeof(size_t);
```

Comment:
Please add a comment what the `7 * sizeof(site_t)` means - otherwise it looks like magic.